product_hazard/rename_model.py

```diff
@@ -0,0 +1,9 @@
+from openupgradelib import openupgrade
+
+
+def rename_model(cr):
+    if openupgrade.table_exists(
+        cr, "beesdoo_product_hazard"
+    ) and not openupgrade.table_exists(cr, "product_hazard"):
+        openupgrade.rename_tables(cr, [("beesdoo_product_hazard", "product_hazard")])
+        openupgrade.rename_models(cr, [("beesdoo.product.hazard", "product.hazard")])
```

Comment:
Could this code not live in beesdoo_product as a pre-migration script? Someone updates beesdoo_product, this script gets run, and subsequently the new dependency `product_hazard` gets installed.

Reply:
Tried it but it doesn't work, the module is installed before the premigration script is ran.